templates/spanish/game/tl/None/common.rpy

```diff
@@ -512,3 +512,26 @@
     old "Position copied to clipboard."
     new "Posición copiada al portapapeles."
 
+    # renpy/common/_errorhandling.rpym:544
+    old "Copy to Clipboard"
+    new "Copiar al portapapeles"
+
+    # renpy/common/_errorhandling.rpym:546
+    old "Copies the traceback.txt file to the clipboard."
+    new "Copia el archivo traceback.txt al portapapeles."
+
+    # renpy/common/_errorhandling.rpym:613
+    old "Copies the errors.txt file to the clipboard."
+    new "Copia el archivo errors.txt al portapapeles."
+
+    # renpy/common/00gltest.rpy:91
+    old "Gamepad"
+    new "Gamepad"
```

Comment:
Thanks, Daniel, I did have a doubt here. I don't use that and don't know how it's called usually in Spanish. I'll put "mando".